File: Praktikum 3 Python/PRAK301-2510817220002-AishaZwarintyanda.py
a, b, c = map(int, input("Masukkan Nilai: ").strip().split())

if a > b:
    a, b = b, a
if b > c :
    b, c = c, b
if a > b:
    a, b = b, a
    
print(f"{a} {b} {c}")

File: Praktikum 3 C/PRAK301-2510817220002-AishaZwarintyanda.c
#include <stdio.h>

int main() {
    int a,b,c,out;

    printf("Masukkan nilai: ");
    scanf("%d %d %d", &a, &b, &c);

    if (a > b){
        out = a;
        a = b;
        b = out;
    }

    if (b > c){
        out = b;
        b = c;
        c = out;
    }

    if (a > b){
        out = a;
        a = b;
        b = out;
    }

    printf("%d %d %d", a, b, c);
    return 0;
}